build-system/pr-check/write-package-jsons.js

```diff
@@ -0,0 +1,87 @@
+/**
+ * Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @fileoverview
+ * Creates package.json files for a given component and AMP version to be
+ * published on npm.
+ */
+
+const [extension, ampVersion] = process.argv.slice(2);
+const {stat, writeFile} = require('fs/promises');
+
+async function writePackageJson(extensionVersion) {
+  try {
+    await stat(`extensions/${extension}/${extensionVersion}`);
+  } catch (e) {
+    console./*OK*/ log(
+      `${extension} ${extensionVersion} : skipping, does not exist`
+    );
+    return;
+  }
+
+  const major = extensionVersion.split('.', 1);
+  const minor = ampVersion.slice(0, 10);
+  const patch = Number(ampVersion.slice(-3)); // npm trims trailing zeroes in patch number, so mimic this in package.json
+  const json = {
+    name: `@ampproject/${extension}`,
+    version: `${major}.${minor}.${patch}`,
+    description: `AMP HTML ${extension} Component`,
```

Comment:
I too am unsure. Should be okay to add this later, since I believe every new push will update the set of keywords.

![image](https://user-images.githubusercontent.com/26553114/117879390-ebfb5680-b274-11eb-96cc-4d2c8d660709.png)

Reply:
SG!